presto-main/src/test/java/com/facebook/presto/operator/scalar/TestDateTimeFunctions.java

```diff
@@ -148,29 +154,102 @@ private static long epochDaysInZone(TimeZoneKey timeZoneKey, long instant)
     public void testLocalTime()
             throws Exception
     {
-        long millis = new LocalTime(session.getStartTime(), DateTimeZone.UTC).getMillisOfDay();
-        functionAssertions.assertFunction("LOCALTIME", TimeType.TIME, toTime(millis));
+        Session localSession = testSessionBuilder()
+                .setSystemProperty("legacy_timestamp", "false")
+                .setTimeZoneKey(TIME_ZONE_KEY)
+                .setStartTime(new DateTime(2017, 3, 1, 10, 0, 0, 0, DateTimeZone.UTC).getMillis())
+                .build();
+        FunctionAssertions localAssertion = new FunctionAssertions(localSession);
```

Comment:
`FunctionsAssertions` are closeable (and contain `LocalQueryRunner`) and should be closed.